src/WopiHost.Core/Infrastructure/UtfString.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace WopiHost.Core.Infrastructure;
+
+/// <summary>
+/// Model for parsing/creating UTF-7 encoded strings.
+/// </summary>
+/// <remarks>the .ToString and implicit string conversion always return the Decoded value</remarks>
+public class UtfString : IParsable<UtfString>
```

Comment:
```suggestion
public class Utf7String : IParsable<UtfString>
```

Reply:
since IMO it will  (eventually) be spec'ed to use Utf8 and this way I would avoid renaming it all...

Reply:
I can live with that